data/reusables/actions/actions-group-concurrency.md

```diff
@@ -1,4 +1,4 @@
-When a concurrent job or workflow is queued, if another job or workflow using the same concurrency group in the repository is in progress, the queued job or workflow will be `pending`. Any pending job or workflow in the concurrency group will be canceled.  This means that there can be at most one running and one pending job in a concurrency group at any time.
+When a concurrent job or workflow is queued, if another job or workflow using the same concurrency group in the repository is in progress, the queued job or workflow will be `pending`. Any pending job or workflow in the same concurrency group, if it exists, will be canceled and the new queued job or workflow will take its place. This means that there can be at most one running and one pending job in a concurrency group at any time.
```

Comment:
I would move the final sentence in this paragraph to be first in the paragraph, and the second sentence should read 'Any existing `pending` job or workflow...'

Reply:
@ctcampbell Thank you for the suggestion! ✨ 

@zachborboacryptofi Once you've had a chance to update your PR with this suggestion, we can get this merged 💛

Reply:
addressed in https://github.com/github/docs/pull/34657/commits/0181615c87ba5eb5687ad60778a33a8075f76cfb